tests/phpunit/integration/Modules/Analytics/AdvancedTrackingTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Class Google\Site_Kit\Tests\Modules\Analytics\AdvancedTrackingTest
+ *
+ * @package   Google\Site_Kit\Tests\Modules\Analytics
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Modules\Analytics;
+
+use Google\Site_Kit\Tests\TestCase;
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking;
+use Google\Site_Kit\Tests\Modules\MockPluginDetector;
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\CF7_Event_List;
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\FormidableForms_Event_List;
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\NinjaForms_Event_List;
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\Woocommerce_Event_List;
+use Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\WPForms_Event_List;
+
+class AdvancedTrackingTest extends TestCase {
+
+	private $supported_plugins;
+
+	private $mock_plugin_detector;
+
+	public function setUp() {
+		parent::setUp();
+
+		if ( ! wp_script_is( 'google_gtagjs' ) ) {
+			wp_enqueue_script( // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
+				'google_gtagjs',
+				'https://www.googletagmanager.com/gtag/js',
+				false,
+				null,
+				false
+			);
+		}
+
+		$this->supported_plugins    = array(
+			'Contact Form 7',
+			'Formidable Forms',
+			'Ninja Forms',
+			'WooCommerce',
+			'WPForms',
+			'WPForms Lite',
+		);
```

Comment:
This should use a similar configuration format like the original, otherwise it's not really accurate.